src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -154,6 +165,168 @@ def coe_add_monoid_hom : finite_measure α →+ measure α :=
+  apply lintegral_mono,
+  exact λ x, ennreal.coe_mono (f_le_g x),
+end
+
+variables [opens_measurable_space α]
+
+lemma test_against_nn_add (μ : finite_measure α) (f₁ f₂ : α →ᵇ ℝ≥0) :
+  μ.test_against_nn (f₁ + f₂) = μ.test_against_nn f₁ + μ.test_against_nn f₂ :=
+begin
+  simp only [←ennreal.coe_eq_coe, bounded_continuous_function.coe_add, ennreal.coe_add,
+             pi.add_apply, test_against_nn_coe_eq],
+  apply lintegral_add,
+  repeat { exact (bounded_continuous_function.nnreal.to_ennreal_comp_measurable _), },
+end
+
+lemma test_against_nn_smul (μ : finite_measure α) (c : ℝ≥0) (f : α →ᵇ ℝ≥0) :
+  μ.test_against_nn (c • f) = c * μ.test_against_nn f :=
+begin
+  simp only [←ennreal.coe_eq_coe, algebra.id.smul_eq_mul, bounded_continuous_function.coe_smul,
+             test_against_nn_coe_eq, ennreal.coe_mul],
+  have key_smul := @lintegral_const_mul _ _ (μ : measure α) c _
+                   (bounded_continuous_function.nnreal.to_ennreal_comp_measurable f),
+  exact key_smul,
+end
+
+lemma test_against_nn_lipschitz_estimate (μ : finite_measure α) (f g : α →ᵇ ℝ≥0) :
+  μ.test_against_nn f ≤ μ.test_against_nn g + (nndist f g) * μ.mass :=
+begin
+  simp [←test_against_nn_const μ (nndist f g), ←test_against_nn_add, ←ennreal.coe_le_coe],
+  apply lintegral_mono,
+  have le_dist : ∀ x, dist (f x) (g x) ≤ (nndist f g),
+  by exact bounded_continuous_function.dist_coe_le_dist,
+  intros x,
+  have le' : f(x) ≤ g(x) + (nndist f g),
+  { apply (nnreal.le_add_nndist (f x) (g x)).trans,
+    rw add_le_add_iff_left,
+    exact dist_le_coe.mp (le_dist x), },
+  have le : (f(x) : ℝ≥0∞) ≤ (g(x) : ℝ≥0∞) + (nndist f g),
+  { rw ←ennreal.coe_add,
+    exact ennreal.coe_mono le', },
+  rwa [coe_nnreal_ennreal_nndist] at le,
+end
+
+lemma test_against_nn_lipschitz (μ : finite_measure α) :
+  lipschitz_with μ.mass (λ (f : α →ᵇ ℝ≥0), μ.test_against_nn f) :=
+begin
+  rw lipschitz_with_iff_dist_le_mul,
+  intros f₁ f₂,
+  suffices : abs (μ.test_against_nn f₁ - μ.test_against_nn f₂ : ℝ) ≤ μ.mass * (dist f₁ f₂),
+  { rwa nnreal.dist_eq, },
+  apply (@abs_le ℝ _ _ _ _ _).mpr,
+  split,
+  { have key' := μ.test_against_nn_lipschitz_estimate f₂ f₁,
+    rw mul_comm at key',
+    suffices : ↑(μ.test_against_nn f₂) ≤ ↑(μ.test_against_nn f₁) + ↑(μ.mass) * dist f₁ f₂,
+    { linarith, },
+    have key := nnreal.coe_mono key',
+    rwa [nnreal.coe_add, nnreal.coe_mul, nndist_comm] at key, },
+  { have key' := μ.test_against_nn_lipschitz_estimate f₁ f₂,
+    rw mul_comm at key',
+    suffices : ↑(μ.test_against_nn f₁) ≤ ↑(μ.test_against_nn f₂) + ↑(μ.mass) * dist f₁ f₂,
+    { linarith, },
+    have key := nnreal.coe_mono key',
+    rwa [nnreal.coe_add, nnreal.coe_mul] at key, },
+end
+
+/-- Finite measures yield elements of the `weak_dual` of bounded continuous nonnegative
+functions via `finite_measure.test_against_nn`, i.e., integration. -/
+def to_weak_dual_of_bounded_continuous_nnreal (μ : finite_measure α) :
```

Comment:
```suggestion
def to_weak_dual_bounded_continuous_nnreal (μ : finite_measure α) :
```
In our naming conventions, `of` refers to an assumption of the theorem or the definition, which is not the case here. So, I think the definition is clearer by just removing the `of`. Another option (which I like even more) is just to call it `to_weak_dual`, because what other weak dual could it be? Whichever way you change the name, it should also be changed in the file-level docstring.